src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -0,0 +1,233 @@
+You can remotely configure the Amplify Logger, enabling you to make changes to your logging levels or user allow list in your deployed applications.
+
+<Callout>
+The logging configurations you set remotely will overwrite the local log level and persist for future app sessions.
+</Callout>
+
+## Set up remote configuration backend resources
+
+<Callout>
+
+You can use the Amplify CLI to [add custom CDK resources](https://docs.amplify.aws/cli/custom/cdk/).  
+
+</Callout>
+
+Below is an example CDK construct that provisions the Amazon CloudWatch, AWS API Gateway, AWS Lambda, and AWS S3 bucket.  The CDK construct also deploys the `remoteloggingconstraints.json` file to the AWS S3 bucket. 
```

Comment:
Smal tweak:

Below is an example CDK construct that provisions the Amazon CloudWatch, AWS API Gateway, AWS Lambda, and AWS S3 bucket. The CDK construct also deploys the remote configuration file to the AWS S3 bucket which you can then update to change the configuration level or user allow list.

Reply:
Description updated per suggestion.